.github/workflows/prod_test.yml

```diff
@@ -0,0 +1,37 @@
+name: End-to-end Production Testing (Nightly)
```

Comment:
Make this more clear.

Run end-to-end test on ros-cross-compile packages published to PIP (Nightly)